app/src/modules/st_common.py

```diff
@@ -197,6 +208,223 @@ def initialize_chatbot(ll_model):
     return cmd
 
 
+def get_yaml_obaas(session_state_json,provider):
+
+    ########## OBAAS VERSION ##########
+    # eureka:
+    #    instance:
+    #       hostname: ${spring.application.name}
+    #       preferIpAddress: true
+    #    client:
+    #      service-url:
+    #        defaultZone: ${eureka.service-url}
+    #    fetch-registry: true
+    #    register-with-eureka: true
+    #    enabled: true
+    vector_table,_= utilities.get_vs_table(session_state_json["rag_params"]["model"],session_state_json["rag_params"]["chunk_size"],session_state_json["rag_params"]["chunk_overlap"],session_state_json["rag_params"]["distance_metric"])
+    instr_context=session_state_json["lm_instr_config"][session_state_json["lm_instr_prompt"]]["prompt"]
+
+    yaml_base=f"""
+server:
+  servlet:
+    context-path: /v1
+spring:
+  datasource:
+    url: ${{spring.datasource.url}}
+    username: ${{spring.datasource.username}}
+    password: ${{spring.datasource.password}}
```

Comment:
I think this code is the code that goes into `application.yaml` if so we should use a pooled connection:

```yaml
  datasource:
    url: ${spring.datasource.url}
    username: ${spring.datasource.username}
    password: ${spring.datasource.password}
    driver-class-name: oracle.jdbc.OracleDriver
    type: oracle.ucp.jdbc.PoolDataSource
    oracleucp:
      connection-factory-class-name: oracle.jdbc.pool.OracleDataSource
      connection-pool-name: UCPConnectionPool
      initial-pool-size: 15
      min-pool-size: 10
      max-pool-size: 30
```

Reply:
checking